pkg/overlay/pack.go

```diff
@@ -34,12 +34,6 @@ import (
 
 var tarEx sync.Mutex
 
-// Container image layers are often tar.gz, however there is nothing in the
-// spec or documentation which standardizes compression params which can cause
-// different layer hashes even for the same tar. So picking compression params
-// that most tooling appears to be using.
-const gzipBlockSize = mutate.GzipBlockSize(256 << 12)
```

Comment:
What size is being used in umoci@v.0.5.0 now?

Any idea why dropping this and bumping skopeo tripped up the test-cases?

Reply:
Found the issue with details https://github.com/opencontainers/umoci/pull/509